js/search.js

```diff
@@ -0,0 +1,64 @@
+ import {getMovies} from "./myAPI.js";
+
+const moviesList = document.getElementById("movies-filter")
+const cinemaMovies = await getMovies();
+
+ 
+//Render some data to DOM from API
+const displayMovies = (movies) => { 
+    let id = cinemaMovies.filter.length;   
+    const htmlString = movies
+        .map((movie) => {
+            return`<a href="page.html/#${id ++}">
```

Comment:
Your way of getting the movie id here is flawed. If I click on "Det stora blå", I should be taken to `movie.html#2` because it's the entry with index 2 in the database array, but your link takes me to `#3`. 
An easy solution is to add the index into the mapping:
`.map((movie, index) => { ... `

Reply:
Klart det ska vara det, måste lära mig att börja räkna från 0 ;)